cmd/otelcol/Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM alpine:3.9 as certs
 RUN apk --update add ca-certificates
+RUN mkdir -m 777 /scratchtmp
 
 FROM scratch
+# Make an empty /tmp directory as the Kubernetes client-go library tries to log to here.
```

Comment:
Is a better fix possible? Can we make client-go library not to do that?

Reply:
Perhaps it is possible, see https://github.com/open-telemetry/opentelemetry-collector/issues/608 - I see this change here as a temporary workaround until someone has time to properly investigate the lib.